Gopkg.toml

```diff
@@ -64,17 +62,46 @@ ignored = ["github.com/rook/rook/.cache","github.com/rook/rook/.work","github.co
 [[constraint]]
   name = "github.com/stretchr/testify"
 
+[[constraint]]
+  name = "k8s.io/utils"
+
+[[constraint]]
+  name = "github.com/rook/operator-kit"
+  branch = "master"
```

Comment:
nit: If using `master`, do we have to explicitely mention it in the with the `branch` constraint?

Reply:
it may not be necessary, but just to be safe